src/results/resultTable.ts

```diff
@@ -233,6 +233,7 @@ export class ResultTable implements IteratorOf<RelTypedValue['value'][]> {
     return new ResultTable({
       relationId: `/${relationId}`,
       table: slicedTable,
```

Comment:
This's not right. But ResultTable doesn't use protobuf metadata yet. It'd be fixed by moving ResultTable to protobuf metadata.